vnext/Shared/CreateModules.h

```diff
@@ -10,7 +10,7 @@
 namespace facebook {
 namespace react {
 
-class MessageQueueueThread;
+class MessageQueueThread;
 
 // This method is to create a unique_ptr of native timing module.
 // @param A MessageQueueThread on which this native module lives.
```

Comment:
What's the motivation for a shared factory function? Is this looking forward to adding the module from shared code?

Reply:
- Eliminate the need to directly access the module-specific header (since we now will have two in MSRN).
- Yes